src/shared/const.ts

```diff
@@ -21,9 +19,8 @@ const USER_NODES_STORAGE_VERSION = '1'
 
 export const USER_NODES_STORAGE_DEFAULT: UserNodesStorage = {
   version: USER_NODES_STORAGE_VERSION,
-  mainnet: A.empty,
-  chaosnet: A.empty,
-  testnet: A.empty
+  mainnet: [],
```

Comment:
BTW: `fp-ts` has marked `A.empty` as deprecated with a note `Use a new [] instead.`, but I could never find such "new" type there. Any idea?

Reply:
not sure but i suppose that means exactly what you have done:
create a new empty array... 
actually seems a pointer to a single instance of an array is unsafe actually 'cuz anyone can push or smth like that to it. i mean this is a mutable object and it seems this is a reason why `fp-ts` deprecated it.

another safely way to use empty array from `fp-ts` as initial value here is to use `ReadOnlyArray`
so with this it will be like this:
```ts
import * as ROA from 'fp-ts/ReadOnlyArray`

export const USER_NODES_STORAGE_DEFAULT: UserNodesStorage = {
  version: USER_NODES_STORAGE_VERSION,
  mainnet: ROA.empty,
  testnet: ROA.empty
}
```
which is `read-only` and safer at type-/runtime-levels